storage/filestore/dir.go

```diff
@@ -18,10 +18,12 @@ import (
 )
 
 const (
-	blobPermission = 0600
-	dirPermission  = 0700
+	blobPermission = 0655
+	dirPermission  = 0755
```

Comment:
Why changing the permissions? What's the reason for other users to mess up with the blob storage on the file system?